ui/admin.html

```diff
@@ -70,7 +69,7 @@
   <script src="vendor/angular/angular-cookies.js"></script>
   <script src="vendor/angular/angular-ui-router.js"></script>
   <script src="vendor/angular/angular-sanitize.js"></script>
-  <script src="vendor/angular-local-storage.min.js"></script>
+  <script src="vendor/ngStorage.min.js"></script>
```

Comment:
Why this change? If it's just a better/newer library for localstorage, can we make it as part of a separate PR that can land sooner?